ui/pages/permissions-connect/permissions-connect.component.js

```diff
@@ -35,7 +35,7 @@ export default class PermissionConnect extends Component {
     approvePermissionsRequest: PropTypes.func.isRequired,
     rejectPermissionsRequest: PropTypes.func.isRequired,
     getRequestAccountTabIds: PropTypes.func.isRequired,
-    accounts: PropTypes.array.isRequired,
+    accounts: PropTypes.arrayOf().isRequired,
```

Comment:
Here you're using `arrayOf` but you're not enforcing the data type of the array items

Reply:
Added the shape in 55991ab71addbf1b4c013d0eafbe913870ab97b9